docs/en/compute/AWS-EMR.md

```diff
@@ -152,16 +152,85 @@ Tuning of Alluxio properties can be done in a few different locations.
 Depending on which service needs tuning, EMR offers different ways of modifying the service
 settings/environment variables.
 
+### Bootstrap Script Usage
+
+```
+Usage: alluxio-emr.sh <root-ufs-uri>
+                             [-b <backup_uri>]
+                             [ -d <alluxio-download-uri>]
+                             [-f <file_uri>]
+                             [-i <journal_backup_uri>]
+                             [-p <delimited_properties>]
+                             [-s <property_delimiter>]
+                             
+alluxio-emr.sh is a script which can be used to bootstrap an AWS EMR cluster
+with Alluxio. It can download and install Alluxio as well as add properties
+specified as arguments to the script.
+  
+By default, if the environment this script executes in does not already contain
+an Alluxio install at /opt/alluxio then it will download, untar, and configure
+the environment at /opt/alluxio. If an install already exists at /opt/alluxio,
+nothing will be installed over it, even if -d is specified.
+  
+If a different Alluxio version is desired, see the -d option.
```

Comment:
maybe instead say if a "specific Alluxio tarball" instead?

Reply:
This should be modified in the bootstrap README. I only pasted the copy into the docs so users wouldn't have to download and view it themselves.

Reply:
agreed. edit here and update?

Reply:
done